### PR TITLE
RHAMT Rebrand Console

### PR DESCRIPTION
**Pull request by RichardHoch**
RHAMT Rebrand Console

Preview: http://file.emea.redhat.com/~rhoch/06252020/Console/

### CODE REVIEW

**Comment by apinnick** [comment]
I spotted a couple diagrams that need to be replaced. Since they have to come from the Jess and team, you'll need to file a diagram request form for both of them. The link is on our migration TW minutes doc. http://file.emea.redhat.com/~rhoch/06252020/Console/#openshift_install

**Comment by apinnick** [comment]
There's a screenshot in "2.2.3. Deploy the MTA Application". Please write that up as text. We shouldn't be using screenshots as a shortcut for the list of fields to fill in.

File: docs/topics_5/known_issues.adoc [hunk 1]
```diff
@@ -25,177 +16,4 @@ At the time of the release the following known issues have been identified as im
 |Web UI
 |After analysis clicking on thee "short reports" link results in a "not found" page
```
Comment (apinnick) [inline docs/topics_5/known_issues.adoc:17]: fix typo

Comment (RichardHoch) [reply docs/topics_5/known_issues.adoc:17]: Fixed.

**Pull request closed without merging**